metrics/error-report/Bugsnag.js

```diff
@@ -1,40 +1,37 @@
 /* eslint-disable no-param-reassign */
 /* eslint-disable import/prefer-default-export */
 /* eslint-disable no-prototype-builtins */
-import ScriptLoader from "../../integrations/ScriptLoader";
-import { MAX_WAIT_FOR_INTEGRATION_LOAD } from "../../utils/constants";
-import { get } from "../../utils/utils";
+import ScriptLoader from '../../integrations/ScriptLoader';
+import { MAX_WAIT_FOR_INTEGRATION_LOAD } from '../../utils/constants';
+import { get } from '../../utils/utils';
 
 // This SDK meta data will be send along with the error for more insight
 const META_DATA = {
   SDK: {
-    name: "JS",
-    installType: "process.module_type",
+    name: 'JS',
+    installType: 'process.module_type',
   },
 };
 
 // This API key token is parsed in the CI pipeline
-const API_KEY = "{{RS_BUGSNAG_API_KEY}}";
+const API_KEY = '{{RS_BUGSNAG_API_KEY}}';
 
 // Errors only from Below SDKs are allowed to reach Bugsnag
 const SDK_FILE_NAMES = [
-  "browser.js",
-  "rudder-analytics.min.js",
-  "rudder-analytics-staging.min.js",
-  "rudder-analytics.js",
+  'browser.js',
+  'rudder-analytics.min.js',
+  'rudder-analytics-staging.min.js',
+  'rudder-analytics.js',
```

Comment:
Let's keep only the `rudder-analytics.min.js` here as I've seen conflicts for `browser.js` and others are not served on production anyway.

Reply:
Addressed.